src/lib/Tests/EventListener/RequestListenerTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Tests\EventListener;
+
+use eZ\Publish\Core\MVC\Symfony\SiteAccess;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Symfony\Component\HttpKernel\KernelEvents;
+use PHPUnit\Framework\TestCase;
+use EzSystems\EzPlatformAdminUi\EventListener\RequestListener;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class RequestListenerTest extends TestCase
+{
+    /** @var RequestListener */
+    private $requestListener;
+
+    /** @var Request */
+    private $request;
+
+    /** @var GetResponseEvent */
+    private $event;
+
+    /** @var HttpKernelInterface|\PHPUnit_Framework_MockObject_MockObject */
```

Comment:
New PHPUnit has namespaces support which we should use.

Reply:
As I see, createMock() return PHPUnit_Framework_MockObject_MockObject